srcpkgs/mesa/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'mesa'
 pkgname=mesa
-version=21.1.6
-revision=2
+version=21.1.7
+revision=1
 wrksrc="mesa-${version}"
```

Comment:
Let's drop this line, `wrksrc="$pkgname-$version"` is the default, so we don't have to overwrite it.

Reply:
I agree, this is vestigial from when the package was named `libGL`

Reply:
I'll get to this in the morning but just to clarify you are recommending removing the wrksrc line entirely? Since the default behavior makes it unnecessary?

Reply:
Yes, we don't need to specify variables that are unnecessary.

Reply:
Okay it's spitting out new errors haha